.github/workflows/code-checks.yml

```diff
@@ -0,0 +1,25 @@
+name: Run Checks on Codebase
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  checks:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Set up Java
+        uses: actions/setup-java@v4
+        with:
+          java-version: '17'
```

Comment:
would the java version specified here become an issue, considering I set up the project with Java 1.8?

Reply:
No, it won't

Reply:
It's perfect then.